### PR TITLE
Change attribute value 

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
float change to int

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5177 https://github.com/pingcap/docs-cn/pull/5182
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -488,13 +488,13 @@ This section introduces configuration items related to the Coprocessor Cache fea
...
 
-- Specifies the largest single push-down calculation result set that can be cached. If the result set of a single push-down calculation returned on the Coprocessor is less than the result set specified by this parameter, the result set is cached. Increasing this value means that more types of push-down requests are cached, but also cause the cache space to be occupied more easily. Note that the size of each push-down calculation result set is generally smaller than the size of the Region. Therefore, it is meaningless to set this value far beyond the size of a Region.
-- Default value: `10.0`
+- Specifies the largest single push-down calculation result set that can be cached. If the result set of a single push-down calculation returned on the Coprocessor is larger than the result set specified by this parameter, the result set is cached. Increasing this value means that more types of push-down requests are cached, but also cause the cache space to be occupied more easily. Note that the size of each push-down calculation result set is generally smaller than the size of the Region. Therefore, it is meaningless to set this value far beyond the size of a Region.
```
Comment (TomShawn) [inline tidb-configuration-file.md:496]:
```suggestion
- Specifies the largest single push-down calculation result set that can be cached. If the result set of a single push-down calculation returned on the Coprocessor is less than the result set specified by this parameter, the result set is cached. Increasing this value means that more types of push-down requests are cached, but also cause the cache space to be occupied more easily. Note that the size of each push-down calculation result set is generally smaller than the size of the Region. Therefore, it is meaningless to set this value far beyond the size of a Region.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by glkappe** [comment]
fix https://github.com/pingcap/docs-cn/pull/5177/files

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -488,13 +488,13 @@ This section introduces configuration items related to the Coprocessor Cache fea
 ### `capacity-mb`
 
 - The total size of the cached data. When the cache space is full, old cache entries are evicted.
-- Default value: `1000.0`
+- Default value: `1000`
```
Comment (TomShawn) [inline tidb-configuration-file.md:491]:
```suggestion
- Default value: `1000.0`
```

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -488,13 +488,13 @@ This section introduces configuration items related to the Coprocessor Cache fea
...
 - Unit: MB
 
 ### `admission-max-result-mb`
 
 - Specifies the largest single push-down calculation result set that can be cached. If the result set of a single push-down calculation returned on the Coprocessor is less than the result set specified by this parameter, the result set is cached. Increasing this value means that more types of push-down requests are cached, but also cause the cache space to be occupied more easily. Note that the size of each push-down calculation result set is generally smaller than the size of the Region. Therefore, it is meaningless to set this value far beyond the size of a Region.
-- Default value: `10.0`
+- Default value: `10`
```
Comment (TomShawn) [inline tidb-configuration-file.md:497]:
```suggestion
- Default value: `10.0`
```

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -488,13 +488,13 @@ This section introduces configuration items related to the Coprocessor Cache fea
 ### `capacity-mb`
 
 - The total size of the cached data. When the cache space is full, old cache entries are evicted.
-- Default value: `1000.0`
+- Default value: `1000`
 - Unit: MB
```
Comment (TomShawn) [inline tidb-configuration-file.md:493]:
```suggestion
Type: Float

```

File: tidb-configuration-file.md [hunk 5]
```diff
@@ -488,13 +488,13 @@ This section introduces configuration items related to the Coprocessor Cache fea
...
 
 ### `admission-max-result-mb`
 
 - Specifies the largest single push-down calculation result set that can be cached. If the result set of a single push-down calculation returned on the Coprocessor is less than the result set specified by this parameter, the result set is cached. Increasing this value means that more types of push-down requests are cached, but also cause the cache space to be occupied more easily. Note that the size of each push-down calculation result set is generally smaller than the size of the Region. Therefore, it is meaningless to set this value far beyond the size of a Region.
-- Default value: `10.0`
+- Default value: `10`
 - Unit: MB
```
Comment (TomShawn) [inline tidb-configuration-file.md:498]:
```suggestion
- Unit: MB
- Type: Float
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**